patterns/2-structured/support-as-a-group.md

```diff
@@ -0,0 +1,86 @@
+
+## Problem
+
+* A popular InnerSource project is orphaned with no one to support it.
+* There isn't an obvious home for it to land.
+
+## Story
+
+A library of UI widgets is used by over 50 projects throughout the company.
+The funding for the team that owns the library runs out and the team disbands.
+At first, no one notices, but after a while whenever someone asks "who owns it" there isn't an answer.
+What will happen next?
+Will new teams shy away from using it?
+Will the project stagnate and linger until its users eventually are forced to move on to something else?
+What a shame if that were to happen to a perfectly good and useful project!
+
+## Context
+
+* Popular InnerSource project.
+* No one is actively supporting it.
+
+## Forces
+
+* No one is assigned by their day job to work on it.
+* Everyone is busy.
+* High cost to migrate off the project.
+
+## Solutions
+
+Call for interested volunteers from anywhere in the company to form a group of [Trusted Committer]s to support the project.
+You may need to reach out to specific individuals based on commit or usage history.
+It is important that there are enough so that the burden on each can be reasonably small.
+
+When forming, this group should identify or create:
+
+* A chat channel where questions and discussion about the project should happen.
+* Clear contributing instructions.
+* Link to both of the above in the project README.
+* A chat channel just for the [Trusted Committer]s.
+
+The group should do its best to manage these aspects of the project:
+
+* **Maintenance**.  If the project is flat-out broken for the standard use case, then fix it.
+Keep the project up-to-date as the dependencies and frameworks it uses continue to evolve.
+* **Onboarding**.  If someone has a question about how to use the project, make sure they get a reasonable answer.
+* **Updates**.  If someone wants to add new feature to the project, give them the design and technical support necessary for them to build it so that it both works for them and is a good addition to the project.
+Review incoming pull requests in a timely manner.
+
+Since this group is comprised of volunteers, it is important to communicate that support is "best effort" only.
+Accordingly this model of support is not well-suited for run-time critical, production projects like live APIs.
+It is better suited for projects that are consumed at build-time like libraries/packages/modules.
+
+## Resulting Context
+
+* There is some "best-effort" support for the InnerSource project.
+
+## Rationale
+
+People generally want to help.
+If there is personal outreach for someone to join as a [Trusted Committer], there are generally a number of people that will say "yes".
+Feeling part of a group and being given some structure and responsibility generally motivates people to try their best, which many times ends up being enough.
+
+## Known Instances
+
+* WellSky
+
+## Status
+
+Structured
```

Comment:
Adding on to this:
We can decide at the very end whether we want to publish this as Initial or Structured.
So don't worry about this too much for now.

Reply:
Hmm. I am reading this for `Structured`:

> Is validated by at least 1 (one) known instance

For the most part this has meant that patterns with one Known Instance will be moved to Structured. But of course we can make our own choice.

Reply:
Oh I see.  Only one required for structured.  I think that this is why I submitted this as `Structured` initially - it's something that I am using at WellSky.

Reply:
Yeah. I think we should stick to the definition as outlined in our Maturity Levels.
That means that the pattern is **Structured**, as it has 1 Known Instance, and also meets all other requirements for that level.

I still think that it might be a flaw in our maturity process, as a single org can publish a pattern straight from zero into our book (with themselves as a known instance).

We (as the patterns working group) should possibly review this in the future but for now let's move on and get this PR merged :)

Reply:
FYI I will also post something similar as my review comment, so that this does not get lost in this thread.